definition/definition.go

```diff
@@ -182,6 +182,102 @@ func (ac *ActivityConfig) OutputMapper() mapper.Mapper {
 }
 
 type loopCfg struct {
+	Condition  string `md:"condition"`
+	IterateOn  string `md:"iterateOn"`
+	Delay      int    `md:"delay"`
+	Accumulate bool   `md:"accumulate"`
+}
+
```

Comment:
I don't think we need 2 loop structs here.  Just keep loop should be good then

Reply:
@skothari-tibco  I still think we can remove loopCfg and just keep loop struct

Reply:
I think if I remove it ; then I won't be able to do `MapToStruct()` as one of the field is `expression` which brings us back to again checking each individual fields ; which was what I was trying to avoid in first place.  Any tips?

Reply:
The tag of `md` I suspend which for activity/trigger metadata using only, which might confuse here by reading the code.  @flogo-oss thoughts?

Reply:
I'll have to think about it a little more.  I believe @skothari-tibco is using so he can call MapToStruct and doesn't have to do the boiler plate validate code.  Maybe after we merge this in I'll take a closer look.  I'd rather get this in sooner than later so we can finish up the flow unit testing.